### ISSUE TITLE
Check failed: status == CUDNN_STATUS_SUCCESS (9 vs. 0)  CUDNN_STATUS_NOT_SUPPORTED

### ISSUE THREAD

**Issue by ergysr** [chuckcho/video-caffe#26]
I am not able to run the sample on the UCF dataset. The error is thrown in cudnn_ndconv_layer.cu:184 (CUDNN_STATUS_NOT_SUPPORTED). All tests in `make runtest` pass (including cudnn tests) and the MNIST example also succeeds. This makes me think the problem is related to the implementation of the ND-convolution layer. Any suggestions how to fix this?

The setup is Ubuntu 16.04 + cuda 7.5 + cuDNN v5 + nvidia gt970m.

> I0628 23:50:24.596218 16220 blocking_queue.cpp:50] Data layer prefetch queue empty
> I0628 23:50:53.321280 16220 solver.cpp:404]     Test net output #0: accuracy/top-1 = 0.015
> I0628 23:50:53.321352 16220 solver.cpp:404]     Test net output #1: accuracy/top-5 = 0.065
> I0628 23:50:53.321363 16220 solver.cpp:404]     Test net output #2: loss = 4.67311 (* 1 = 4.67311 loss)
> I0628 23:50:53.321367 16220 solver.cpp:337] Iteration 0, Testing net (#1)
> I0628 23:51:22.838021 16220 solver.cpp:404]     Test net output #0: accuracy/top-1 = 0
> I0628 23:51:22.838043 16220 solver.cpp:404]     Test net output #1: accuracy/top-5 = 0.045
> I0628 23:51:22.838052 16220 solver.cpp:404]     Test net output #2: loss = 4.68178 (* 1 = 4.68178 loss)
> F0628 23:51:23.015211 16220 cudnn_ndconv_layer.cu:184] Check failed: status == CUDNN_STATUS_SUCCESS (9 vs. 0)  CUDNN_STATUS_NOT_SUPPORTED
> *** Check failure stack trace: ***
>     @     0x7f32a20195cd  google::LogMessage::Fail()
>     @     0x7f32a201b433  google::LogMessage::SendToLog()
>     @     0x7f32a201915b  google::LogMessage::Flush()
>     @     0x7f32a201be1e  google::LogMessageFatal::~LogMessageFatal()
>     @     0x7f32a286572b  caffe::CudnnNdConvolutionLayer<>::Backward_gpu()
>     @     0x7f32a281049b  caffe::Net<>::BackwardFromTo()
>     @     0x7f32a28104ff  caffe::Net<>::Backward()
>     @     0x7f32a263fdec  caffe::Solver<>::Step()
>     @     0x7f32a2640879  caffe::Solver<>::Solve()
>     @           0x40b487  train()
>     @           0x4075a8  main
>     @     0x7f32a0acf830  __libc_start_main
>     @           0x407d19  _start
>     @              (nil)  (unknown)
> Aborted (core dumped)

Thanks!

**Comment by ergysr** [comment]
I was getting this error because I modified one of the following lines in Makefile.config. Once I reverted my changes I was able to build the project.

> CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
> 		-gencode arch=compute_20,code=sm_21 \
> 		-gencode arch=compute_30,code=sm_30 \
> 		-gencode arch=compute_35,code=sm_35 \
> 		-gencode arch=compute_50,code=sm_50 \
> -gencode arch=compute_50,code=compute_50

**Comment by chuckcho** [comment]
Good that you fixed it, @ergysr
